### PR TITLE
40 cli data versioning and help instructions

### LINKED ISSUE #40: CLI data versioning and help instructions
### Description
Add a command to the CLI that allows the admin to see the version of the data currently used on the interface (probably by date of last update / date of last check). Either separately or together, write up some instructions for the help command of the CLI, including a description of the CLI and of the other commands that can be called. 

### To-do
* [x] Help command description of CLI and description of other commands
* [x] Allow CLI user to see version of data on interface

### PR DESCRIPTION
**Pull request by keyman015**
### Description 

The CLI tool has been updated with an input menu of sorts to allow users to directly modify the database versions and generate new databases. The README documentation has also been update to ensure users know how to use the script.

### To-dos
* [x] ~~Help command description of CLI and description of other commands~~ Instead used an inbuilt menu to reduce confusion
* [x] Allow CLI user to see version of data on interface

### Misc
Closes #40

### CODE REVIEW

File: src/cli/cli.py [hunk 1]
```diff
@@ -1,41 +1,133 @@
...
+            typer.echo(f"Database was created at '{db_date}'.")
+
+        elif choice == "U":
+            typer.echo("Updating the database...")
+            update_db()
+            typer.echo("Database updated successfully!")
+
+        elif choice == "M":
+            typer.echo("Entering Database Version Control menu...")
+            modify_db_menu()  # Placeholder for the actual function
```
Comment (paulmmaingi) [inline src/cli/cli.py:71]: Placeholder?

File: src/cli/cli.py [hunk 2]
```diff
@@ -1,41 +1,133 @@
...
+    typer.echo(f"Current database in use: {current_db}\n")
+    typer.echo("Available databases:")
+
+    # List all available databases
+    for i, db in enumerate(dbs, start=1):
+        typer.echo(f"{i}. {db['filename']} (Created: {db['created_at']})")
+
+    typer.echo("\nOptions:")
+    typer.echo("D = Delete a database")
+    typer.echo("U = Update the current database in use")
```
Comment (paulmmaingi) [inline src/cli/cli.py:95]: I think maybe using a different letter since U in previous menu is update. (C - change?)

**Review by paulmmaingi — COMMENTED** [review]
Looks good!! 

Perhaps later we can also add a Cron job to update the database automatically.

File: src/cli/cli.py (hunk 1 above)
Comment (keyman015) [reply src/cli/cli.py]: Sorry, that was a remnant from the old code.

File: src/cli/cli.py (hunk 2 above)
Comment (keyman015) [reply src/cli/cli.py]: Yes, good pickup. Has been fixed now.

**Review by paulmmaingi — APPROVED** [review]
Awesome! LGTM

**Pull request merged**